docs/QSG/docker.md

```diff
@@ -37,91 +37,59 @@ Virtual Docker networking is still in development. The following are known issue
 ### Hardware and software configuration used in this QSG
 - Server Board: Wolf Pass
 - CPU: Cascade Lake (2)
-- OS: Ubuntu 20.0.4LTE. freshly Installed
+- OS: Ubuntu 20.0.4LTE. fresh Install
 - Memory: 192GB 
-- PMEM:(12)x16GB
+- PMEM:(12)x16GB (not required for this QSG)
 
 ## Configuring Hugepages
 Additional information in Hugepages is located in the administration portion of this guide <need link> and at (kernel.org)[https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt]
 	
 ### Determining Huge pages
-The number of huge pages allocated could be checked with the following command: 
-- Ubuntu- ```$ sysctl vm.nr_hugepages```
-
 The default size of a huge page, the number of available huge pages, etc. could be found with the following command: 
 - Ubuntu- ```$ cat /proc/meminfo | grep -e "^Huge" ```
+If the pages are not enabled, you will see something like this:
+```
+HugePages_Total:       0
+HugePages_Free:        0
+HugePages_Rsvd:        0
+HugePages_Surp:        0
+Hugepagesize:       2048 kB
+Hugetlb:               0 kB
+```
 
-@@@ Not even sure what this means-The platform was tested and validated with the [rockylinux/rockylinux:8.4](https://hub.docker.com/r/rockylinux/rockylinux) and [centos:centos8](https://hub.docker.com/_/centos) official docker images.  However other RHEL-like distributions should be supported.
-
-@@@ Is this a warning about rocky and huge pages or daos with hugepages on rocky or??? 
-!!! warning
-    Some distributions are not yet well supported such as [rockylinux/rockylinux:8.5](https://hub.docker.com/r/rockylinux/rockylinux): issue with the management of hugepages with the [spdk](https://spdk.io/) library.
+#### Note: 
+Most Linux docker distributions should support hugepages. However only the following have been validated
```

Comment:
In fact the huge pages are supported by all containerized Linux distributions. It the support of huge pages with DAOS and its dependencies which are not working for some Linux distribution.